### DISCUSSION TITLE
Is there any way to see which user is making the request in the logs?

### DISCUSSION THREAD

**Discussion by Verzach3** [pocketbase/pocketbase#671] [Q&A] [answered]
I didn't find any info about that in the docs, so i was wondering if it is posible.

**Comment by ganigeorgiev** [comment, marked as answer]
No, currently only the request IP is stored alongside some meta information whether the client is an admin, user or guest.

I have it in my todo and in the future I may consider adding it but that will be after the users refactoring from #376, since we'll have to store the related auth collection id too.
